### PR TITLE
Automation example to change a source on LG TV

### PR DESCRIPTION
**Pull request by artemkovalyov**
It's not completely clear from the docs what method to use to launch TV apps  although they are listed in the sources after the integration is completed. This simple method works pretty well on my TV.

## Proposed change

Added and example for a frequent use-case

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new section on "Switching source with automation" for the webOS TV integration, providing a practical example for users to automate switching to YouTube when the TV is turned on.
- **Bug Fixes**
	- Corrected minor spelling errors in the new section title.

### CODE REVIEW

File: source/_integrations/webostv.markdown [hunk 1]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
 
 It is possible to select which sources will be available to the media player. When the TV is powered on press the **CONFIGURE** button in the {% term integration %} card and select the sources to enable. If you don't select any source the media player will offer all of the sources of the TV.
 
+### Switching source with automation
+
+Imagine you want your LG TV to automatically switch to a specific source when it turns on. Below is a simple automation example that launches `YouTube` after the TV is switched on.
+It leverages `select_source` action from the [Media player](/integrations/media_player/) integration to launch a specific app installed on your LG TV.
+To find available sources for your TV
+
+1. Go to Developer Tools -> States
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]:
```suggestion
1. Go to {% my developer_states title="**Developer Tools** > **States**" %}.
```

File: source/_integrations/webostv.markdown [hunk 2]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
...
+2. Find your TV's media_player entity
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]:
```suggestion
2. Find your TV's media_player entity.
```

File: source/_integrations/webostv.markdown [hunk 3]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
...
+3. Look for the `source_list` attribute, which contains all available sources
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]:
```suggestion
3. Look for the `source_list` attribute which contains all available sources.
```

File: source/_integrations/webostv.markdown [hunk 4]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
...
+  - action: media_player.select_source
+    metadata: {}
+    data:
+      source: YouTube
+    target:
+      device_id: <TV DEVICE ID>
+mode: single
+```
+
+The automation can be created entirely through the Home Assistant UI. When setting it up, you'll only need to manually enter the source name (e.g., "YouTube") in the action configuration.
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]:
```suggestion
The automation can be created entirely through the Home Assistant UI. When setting it up, you'll only need to manually enter the source name (for example, "YouTube") in the action configuration.
```

File: source/_integrations/webostv.markdown [hunk 5]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
...
+  - action: media_player.select_source
+    metadata: {}
+    data:
+      source: YouTube
+    target:
+      device_id: <TV DEVICE ID>
+mode: single
+```
+
+The automation can be created entirely through the Home Assistant UI. When setting it up, you'll only need to manually enter the source name (for example, "YouTube") in the action configuration.
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]: would it make sense to place this information before the YAML example?

File: source/_integrations/webostv.markdown [hunk 6]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
...
+To find available sources for your TV
+
+1. Go to {% my developer_states title="**Developer Tools** > **States**" %}.
+2. Find your TV's media_player entity.
+3. Look for the `source_list` attribute which contains all available sources.
+   
+{% tip %}
+Source list example: `source_list: ARD Mediathek, Apps, HDMI 1, Home Dashboard, JBL Bar 1300, Media Player, Netflix, Prime Video, Public Value, Spotify - Music and Podcasts, Timer, Web Browser, YouTube, ZDFmediathek`
+{% endtip %} 
+
```
Comment (artemkovalyov) [inline source/_integrations/webostv.markdown:83]: The automation can be created entirely through the Home Assistant UI. When setting it up, you'll only need to manually enter the source name (for example, "YouTube") in the action configuration. Below is the YAML code generated as a result:

File: source/_integrations/webostv.markdown (hunk 5 above)
Comment (artemkovalyov) [inline source/_integrations/webostv.markdown]:
```suggestion


```

File: source/_integrations/webostv.markdown (hunk 6 above)
Comment (artemkovalyov) [inline source/_integrations/webostv.markdown:83]:
```suggestion

The automation can be created entirely through the Home Assistant UI. When setting it up, you'll only need to manually enter the source name (for example, "YouTube") in the action configuration. Below is the YAML code generated as a result:

```

File: source/_integrations/webostv.markdown [hunk 7]
```diff
@@ -66,6 +66,44 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
...
+    metadata: {}
+    data:
+      source: YouTube
+    target:
+      device_id: <TV DEVICE ID>
+mode: single
+```
+
+
+
```
Comment (artemkovalyov) [inline source/_integrations/webostv.markdown]:
```suggestion
```

File: source/_integrations/webostv.markdown (hunk 5 above)
Comment (artemkovalyov) [reply source/_integrations/webostv.markdown]: @c0ffeeca7, I changed as you suggested. Give it another look.

File: source/_integrations/webostv.markdown [hunk 8]
```diff
@@ -66,6 +66,42 @@ Any other [actions](/docs/automation/action/) to power on the device can be conf
 
 It is possible to select which sources will be available to the media player. When the TV is powered on press the **CONFIGURE** button in the {% term integration %} card and select the sources to enable. If you don't select any source the media player will offer all of the sources of the TV.
 
+### Switching source with automation
+
+Imagine you want your LG TV to automatically switch to a specific source when it turns on. Below is a simple automation example that launches `YouTube` after the TV is switched on.
+It leverages `select_source` action from the [Media player](/integrations/media_player/) integration to launch a specific app installed on your LG TV.
+To find available sources for your TV
```
Comment (artemkovalyov) [inline source/_integrations/webostv.markdown:74]:
```suggestion

To find available sources for your TV
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @artemkovalyov :+1:

**Pull request merged**